models/UserManager.php

```diff
@@ -0,0 +1,315 @@
+    function get_modos(){
+         $req = $this->db->query("
+            SELECT * FROM admins
+        ");
+
+        $results = [];
+        while($rows = $req->fetchObject()){
+            $results[] = $rows;
+        }
+        
+        return $results;
+    }
+
+    // Fonction qui vérifie qu'un utilisateur est bien modérateur ou admin
+    function is_modo($email,$token){
+        $tableau = [
+            'email' =>  $email,
+            'token' =>  $token
+        ];
+        $sql = "SELECT * FROM admins WHERE email=:email AND token=:token";
+        $req= $this->db->prepare($sql);
+        $req->execute($tableau);
+        
+        return $req->rowCount($sql);
+    }
+
+    // Fonction permettant la mise à jour du mot de passe 
+    function update_password($password){
+         $tableau = [
+            'password'  =>  sha1($password),
+            'session'   =>  $_SESSION['admin']
+        ];
+
+        $sql = "UPDATE admins SET password = :password WHERE email=:session";
+        $req = $this->db->prepare($sql);
+        $req->execute($tableau);
+
+    }
+
+    // Fonction permettant de récupérer la liste des administrateurs
+    function get_user(){
+        $req = $this->db->query("
+            SELECT * FROM admins WHERE email='{$_SESSION['admin']}';
+        ");
+
+        $result = $req->fetchObject();
+        
+        return $result;
+    }
+    
+    function user_verification()
+    {
+           if(filter_has_var(INPUT_POST, 'submit')){
+                    if(filter_has_var(INPUT_POST, 'email')){
+
+                        $email = filter_var(htmlspecialchars(filter_input(INPUT_POST, 'email', FILTER_VALIDATE_EMAIL)));
+                    }
+                    if(filter_has_var(INPUT_POST, 'password')){
+                        $password = filter_var(htmlspecialchars(filter_input(INPUT_POST, 'password')));
+
+                    }
+                    $errors = [];
+
+                    if(empty($email) || empty($password)){
+                        $errors['empty'] = "Tous les champs n'ont pas été remplis!";
+                    }else if(Usermanager::is_admin($email,$password) == 0){
+                        $errors['exist']  = "Cet administrateur n'existe pas";
+                    }
+                    
+                    if(!empty($errors)){
+                        ?>
+                        <div class="card red">
+                            <div class="card-content white-text">
+                                <?php
+                                    foreach($errors as $error){
+                                        echo $error;
+
+                                    }
+                                ?>
+                            </div>
+                        </div>
+                        <?php
+                    }else{
+                        $_SESSION['admin'] = $email;
+                        header("Location:/dashboard");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The use of function header() is discouraged](https://app.codacy.com/manual/JeromeLacquemant/projet-5/pullRequest?prid=5379676)